tests/cross/src/test/scala/tests/pc/MacroCompletionSuite.scala

```diff
@@ -136,7 +136,7 @@ object MacroCompletionSuite extends BaseCompletionSuite {
     compat = Map(
       // NOTE(olafur): the presentation compiler returns empty results here in 2.12.9 and 2.13.0
       "2.12.9" -> "",
-      "2.13" -> ""
+      "2.13.0" -> ""
```

Comment:
Do I understand correctly that the 2.13 regressions for macros have been fixed in 2.13.1?

Reply:
yes, and also 2.12.10 (we hadn't noticed it at the time)